bin/MadGraph5_aMCatNLO/cards/production/13TeV/MajoranaNeutrinoToTriLeptons/MajoranaNeutrinoToMuMuMu_M-1000_5f_LO/MajoranaNeutrinoToMuMuMu_M-1000_5f_LO_proc_card.dat

```diff
@@ -0,0 +1,42 @@
+#************************************************************
+#*                     MadGraph5_aMC@NLO                    *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 2.1.0                 2014-02-21         *
+#*                                                          *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph5_aMC@NLO         *
+#*                                                          *
+#*     run as ./bin/mg5_aMC  filename                       *
+#*                                                          *
+#************************************************************
+
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set gauge unitary
+set complex_mass_scheme False
+import model sm
```

Comment:
I suggest to remove the lines between "import model sm" and "define vl~ = ve~ vm~ vt~" for better clarity